### PR TITLE
Sync the configuration alternatives with the new configurations of the experiment

### PR DESCRIPTION
**Pull request by ipa-nhg**
Related to : https://github.com/rosin-project/rosin-experiments/commit/629ccd33e8f818acd0b7f9879f3eac5c6691494c

The new launch files for the alternatives are also available under: https://github.com/rosin-project/metacontrol_move_base_configurations/pull/9

### CODE REVIEW

**Comment by ipa-nhg** [comment]
ping @chcorbato , I am not sure if this changes are useful but  at least now the yaml file is consistent with the current experiment configuration and the structure of the reasoner repositories.

I couldn't test it my self because I don't have currently a proper python3 environment on my machine, but in theory you should:



```
(copy-paste from the docu from this repo)
  mkdir -p ~/mros1_reasoner_ws/src
  cd mros1_reasoner_ws
  cd ~/mros1_reasoner_ws
  wstool init ~/mros1_reasoner_ws/src https://raw.githubusercontent.com/tud-cor/mc_mros_reasoner/master/mros1_reasoner/mros1_reasoner.rosinstall
  rosdep install --from-paths ~/mros1_reasoner_ws/src -y -i -r
```

and then update the branches and clone the .rossystem files:

```
cd ~/mros1_reasoner_ws/src/mros1_reasoner
git pull --rebase https://github.com/ipa-nhg/mc_mros_reasoner UpdateMetacontrolExperimentConfigFile
cd ~/mros1_reasoner_ws/src/mc_mdl_tomasys
git checkout master
cd   ~/mros1_reasoner_ws/src
git clone https://github.com/rosin-project/metacontrol_move_base_configurations
cd  ~/mros1_reasoner_ws
catkin build
source ~/mros1_reasoner_ws/devel/setup.bash
```

Lastly, you have to call the python script to generate the Owl, as argument you have to give the path to the config file:
```
rosrun mros1_reasoner rosmodel2owl.py ~/mros1_reasoner_ws/src/mros1_reasoner/mros1_reasoner/config/nav_config.yaml 
```

Good luck!

**Comment by chcorbato** [comment]
I am fwding this to @marioney because he has more control over the experimental setup.
@marioney if you prefer me to test it (can be a good idea), could you please point me to the updated instructions to run the experiments?

File: mros1_reasoner/config/nav_config.yaml [hunk 1]
```diff
@@ -1,11 +1,11 @@
 ontology_pkg: mc_mdl_tomasys
-ontology_path: tomasys.owl
+ontology_path: owl/tomasys.owl
 # where is ros_navigation.owl ??
-domain_ontology_pkg: mros1_reasoner
-domain_ontology_path: owl/ros_navigation.owl
+domain_ontology_pkg: mc_mdl_tomasys
+domain_ontology_path: owl/navigation_domain.owl
```
Comment (marioney) [inline mros1_reasoner/config/nav_config.yaml:5]: @chcorbato is this OK? I think goes with the changes you made to the ontology but I'm not sure

**Comment by marioney** [comment]
> @marioney if you prefer me to test it (can be a good idea), could you please point me to the updated instructions to run the experiments?

You can use this one https://github.com/rosin-project/metacontrol_experiments

**Comment by Rezenders** [comment]
I will close this PR since it has been open for more than 2 years and it has a lot of conflicts.

**Pull request closed without merging**